### PR TITLE
Update more OWNERS file. Prefer WGs, instead of individuals.

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: extensions/amp-smartlinks/OWNERS [hunk 1]
```diff
@@ -8,7 +8,7 @@
         {name: 'philwinchester'},
```
Comment (zhouyx) [inline extensions/amp-smartlinks/OWNERS:8]: Since you're on this, let's add `requestReviews: false`

File: extensions/amp-sticky-ad/OWNERS [hunk 2]
```diff
@@ -4,7 +4,7 @@
 {
   rules: [
     {
-      owners: [{name: 'zhouyx'}],
+      owners: [{name: 'ampproject/wg-ads-reviewers'}],
```
Comment (zhouyx) [inline extensions/amp-sticky-ad/OWNERS:7]: I would like to have myself added as an owner here please.

Comment (lannka) [reply extensions/amp-sticky-ad/OWNERS]: done

File: extensions/amp-smartlinks/OWNERS (hunk 1 above)
Comment (lannka) [reply extensions/amp-smartlinks/OWNERS]: done

File: extensions/amp-sticky-ad/OWNERS (hunk 2 above)
Comment (zhouyx) [reply extensions/amp-sticky-ad/OWNERS]: Thanks

**Pull request merged**